controllers/complianceeventsapi/server.go

```diff
@@ -0,0 +1,334 @@
+package complianceeventsapi
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha1"
+	"database/sql"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/doug-martin/goqu/v9"
+	_ "github.com/doug-martin/goqu/v9/dialect/postgres" // blank import the dialect driver
+	_ "github.com/lib/pq"
+)
+
+var (
+	clusterKeyCache      sync.Map
+	parentPolicyKeyCache sync.Map
+	policyKeyCache       sync.Map
+)
+
+type databaseAPIServer struct {
+	Lock      *sync.Mutex
+	server    *http.Server
+	isRunning bool
+}
+
+// Start starts the http server. If it is already running, it has no effect.
+func (s *databaseAPIServer) Start(dbURL string) error {
+	s.Lock.Lock()
+	defer s.Lock.Unlock()
+
+	if s.isRunning {
+		return nil
+	}
+
+	mux := http.NewServeMux()
+
+	s.server = &http.Server{
+		Addr:    ":5480",
```

Comment:
This should be configurable. It should also default to listen only on localhost as well.